blocks/pagination/pagination.js

```diff
@@ -1,102 +1,28 @@
 import { getTextLabel, createElement } from '../../scripts/common.js';
```

Comment:
Also I wonder what is the sense of this block? :P 
@jonatan-lledo-netcentric 
And @santi-homps 
I think you worked on this block, is this block even suppose to be used as a block?

Reply:
Not really, this is just for the search-results template... With the understanding we have now I would say this could be refactored and taken out of the word document.

Reply:
I don't have the access to SharePoint. Also, technical debt is is not in the scope of the ticket, if it were then the ticket would be refined with more story points. There were also mentions of including real pagination in the second phase. Therefore I suggest skipping this and focusing on delivering the feature